backend/databasemodel/model.sql

```diff
@@ -1543,6 +1582,30 @@ COMMENT ON COLUMN kooste.metadata.datasource IS E'You can get this information f
 ALTER TABLE kooste.metadata OWNER TO tarmo_admin;
 -- ddl-end --
 
+INSERT INTO kooste.metadata (
+    codes_tarmo_hiihto,
```

Comment:
Here, we don't have enough information to set the tarmo category string.

If metadata contains rows "laavut_majat_ruokailu", "vesilla_ulkoilu" etc., how does our code know that "laavut_majat_ruokailu" corresponds to "Laavut, majat, ruokailu" unicode string, and "vesilla_ulkoilu" should be translated to "Vesillä ulkoilu" unicode string?

Reply:
So we have a few different ways to do the mapping in my opinion.

Looks like all the other tables (not lipas) already contain the correct tarmo category?

In this case, perhaps this lipas specific code -> category mapping should be saved in lipas.metadata table?

Reply:
Perhaps lipas.metadata table should have a json field `category_by_code` that contains json like 
```
{"Hiihto": [4402,4440,4630,4640] ,
...}
```

This way, our lipas importer knows which tarmo category should be saved, based on the type code?

Reply:
Yes, eventually lipas tables are the only ones in which we need some additional information in order to determine the tarmo_category. For sure, a better place for these type_code lists would be in lipas.metadata table. I will change this according to your suggestion.

Reply:
Yeah that's good. Of course, still our UI will depend on the fact that the strings must be written exactly the correct way in unicode, but we will have to live with that.

Another option would be just to have some shortened categories here in English ("skiing") or Finnish ("hiihto"), and then have the actual user-facing strings in the UI. But perhaps we want to have in the database exactly the same strings that the UI shows to the user.

Reply:
Yes, let's use exactly same strings for now.